keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -262,36 +263,21 @@ Upgrade each node of the control plane to 1.22 with the following updates:
 - CCM deployed with `--enable-leader-migration`
 - CCM has its `--cloud-provider` set to the correct cloud provider
 
-After upgrade, CCM will run `route-controller`, `service-controller`, `cloud-node-controller`,
-and `cloud-nodelifecycle-controller`. The Leader Migration support will ensure CCM cleanly take out these controllers
-during the control plane upgrade.
+The provided default configuration in version 1.22 should be exactly the same as that of 1.21, assuming there are no
```

Comment:
For it to be exactly the same we will need to cherry pick the default and support for wildcard to 1.21. I like that idea but not sure if the release managers will approve the cherry pick. Might be better to just state that the proper default will be supported starting in 1.22. We can then fix this if the release managers approve the cherry pick.